packages/paste-core/components/sidebar/stories/full-composition/segment.stories.tsx

```diff
@@ -53,7 +53,7 @@ export const Segment: StoryFn = () => {
   const mainContentSkipLinkID = useUID();
 
   return (
-    <Box>
+    <Box minWidth="1000px">
```

Comment:
What's this change for?

Reply:
It basically forces this https://deploy-preview-3346--paste-docs.netlify.app/patterns/navigation#segment not to collapse when Iframed in 
<img width="741" alt="image" src="https://github.com/twilio-labs/paste/assets/368249/5ab934d8-00e6-4ce6-acd2-84e08ad87c1f">